services/traction/tests/test_utils.py

```diff
@@ -0,0 +1,51 @@
+import string
+import random
+import os
+import json
+
+from httpx import AsyncClient
+
+
+def random_string(prefix: str, length: int = 10) -> str:
+    letters = string.ascii_lowercase
+    return prefix.join(random.choice(letters) for i in range(length))
+
+
+async def innkeeper_auth(test_client: AsyncClient) -> str:
+    username = os.environ.get("TRACTION_API_ADMIN_USER", "innkeeper")
+    password = os.environ.get("TRACTION_API_ADMIN_KEY", "change-me")
+    data = {"username": username, "password": password}
+    token_resp = await test_client.post("/innkeeper/token", data=data)
+    assert token_resp.status_code == 200, token_resp.content
+    token_content = json.loads(token_resp.content)
+    bearer_token = "Bearer " + token_content["access_token"]
+
+    return bearer_token
```

Comment:
Can these (innkeeper_auth and tenant_auth) be fixtures instead. That also means they can be scoped to 'module' so one auth token could get reused by multiple tests. 

If they aren't used by the test they don't run, so no downside.

Reply:
I don't think there's any benefit to making these fixtures, it makes the tests more opaque (not obvious that there is an auth process).  For tenants, we can't auth until we actually have a tenant, and we may have multiple tenants within a test.